reactExercise/solutions/zihe/ex8/src/NameInfo.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import Spinner from './spinner';
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js';
+import { Pie } from 'react-chartjs-2';
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+function NameInfo() {
+    const [name, setName] = useState('');
+    const [age, setAge] = useState(null);
+    const [gender, setGender] = useState(null);
+    const [nation, setNation] = useState([[], [], [], [], []]);
+    const [loading, setloading] = useState(false);
```

Comment:
boolean变量一般会喜欢用 isXXXX  来起头
比如说 isLoading,  isFull 
```suggestion
    const [isLoading, setILoading] = useState(false);
```